api/tests/test_pcluster_api_handler.py

```diff
@@ -0,0 +1,20 @@
+import pytest
```

Comment:
todo: please add to the readme how to run Python test as we'll need it to setup the test pipeline